content/blog/mlops-–-deploying-an-ml-model-in-greenlake-platform-mlops-service.md

```diff
@@ -0,0 +1,318 @@
+---
+title: ML Ops – Deploying an ML model in HPE GreenLake Platform ML Ops service
+date: 2022-08-08T07:48:49.530Z
+author: Thirukkannan M
+authorimage: /img/Avatar1.svg
+tags:
+  - hpe-greenlake
+---
+**Overview**
+
+HPE GreenLake Central is an advanced software-as-a-service platform that provides you with a consistent cloud experience for all your applications and data—on-premises or off-premises. It provides you with insights and controls to manage your hybrid IT estate, complementing your use of public clouds and data centers. HPE GreenLake Central gives you the ability to choose where and how to place your workloads and data, and—through the services you purchase—enables you to monitor security, capacity, resource utilization, and costs.
```

Comment:
Hi there. The hyphens you have before and after "through the services you purchase" should have spaces between them and the words before and after them. At least, they ought to look as though they do in the final product :-).